network/src/commonMain/kotlin/com/wire/kalium/network/api/base/unbound/acme/ACMEApi.kt

```diff
@@ -40,7 +40,7 @@ interface ACMEApi {
     suspend fun getACMENonce(url: String): NetworkResponse<String>
     suspend fun sendACMERequest(url: String, body: ByteArray? = null): NetworkResponse<ACMEResponse>
     suspend fun sendChallengeRequest(url: String, body: ByteArray): NetworkResponse<ChallengeResponse>
-    suspend fun getACMEFederation(baseUrl: String): NetworkResponse<ByteArray>
+    suspend fun getACMEFederation(baseUrl: String): NetworkResponse<String>
```

Comment:
We could maybe use a `value class`?

Not 100% sure what's gonna be the use of this data, but it would make sure we don't pass Strings where we shouldn't.

```kotlin
val certificateData: String = ""
val user = User(name = certificateData) // Compiler sees no evil
```

But with a value class, the Compiler won't like it.

```kotlin
@JvmInline
value class CertificateData(val value: String)
```